src/ui/public/kuery/functions/is.js

```diff
@@ -33,13 +33,12 @@ export function toElasticsearchQuery(node, indexPattern) {
     return { match_all: {} };
   }
   else if (fieldName === '*' && value !== '*') {
-    const userQuery = String(value);
-    const query = isDoubleQuoted(userQuery) ? userQuery : `"${userQuery}"`;
-
     return {
-      simple_query_string: {
-        query,
-        all_fields: true
+      multi_match: {
+        query: value,
+        fields: ['*'],
```

Comment:
There are 2 potential reasons I can think of to hold off on that for now:

* `lenient` isn't necessary, and is perhaps undesirable when querying against a single field
* Passing the field name straight through to multi_match would allow people to use leading wildcards and boosting which I don't think we should support just yet